tests/integration/ioredis.Dockerfile

```diff
@@ -10,14 +10,15 @@ WORKDIR /app
 RUN apt update -y && apt install -y git
 
 # Clone ioredis v5.2.3
-RUN git clone --branch v5.2.3 https://github.com/luin/ioredis
+#RUN git clone --branch v5.2.3 https://github.com/luin/ioredis
+RUN git clone https://github.com/luin/ioredis
 
 WORKDIR /app/ioredis
 
 RUN npm install
 
 # This is required until https://github.com/luin/ioredis/issues/1671 is resolved.
-ADD .patch_ioredis.sh patch_ioredis.sh
+#ADD .patch_ioredis.sh patch_ioredis.sh
```

Comment:
what's up with this?

Reply:
we don't need a specific branch here to pass the tests, and (I need to remove this comment out), and we need to have the fixes that I made to ioredis. So there is no reason for checking out like this any more.